gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-core/src/main/java/io/gravitee/am/gateway/handler/vertx/auth/webauthn/store/RepositoryCredentialStore.java

```diff
@@ -53,10 +54,13 @@ public class RepositoryCredentialStore {
     @Autowired
     private Domain domain;
 
```

Comment:
I think we should set a default value or -1 for no limit. I’m thinking about the breaking changes for users not using the gravitee yml file to configure Gravitee but rather using docker env values or kubernetes deployment. WDYT ?

Reply:
ow, I thought gravitee.yaml is a part of each deployment bundle - then the default value is set there (20)
But in such case I can add @Value("${user.webAuthn.maxAllowCredentials:**20**}") or whatever number.

-1/no-value is an option of course but is it really needed? who has more than 20 credentials and uses those beyond the threshold?

Reply:
I'm thinking mainly about the customer who has raised this ticket, I'm afraid that some end-users will be blocked after the upgrade of the platform. 

I would prefer to set no limit especially for a maintenance release. We can think about a default value (20) for the next major version where this option will be accessible in the UI / Rest API.

Reply:
Ok. I commented-out gravitee.yml property and set default value to -1. In such case, all credentials are fetched.